src/quo/components/links/internal_link_card/user/view.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo.components.links.internal-link-card.user.view
+  (:require
+    [quo.components.icon :as icon]
+    [quo.components.links.internal-link-card.user.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]
+    [react-native.linear-gradient :as linear-gradient]))
+
+(defn- subtitle-comp
+  [subtitle emojis]
+  [rn/view
+   [text/text
+    {:size                :paragraph-2
+     :number-of-lines     2
+     :accessibility-label :subtitle
+     :style               {:margin-bottom 12}} subtitle]
+   [rn/view {:flex-direction :row}
```

Comment:
Guideline *Always add styles inside the :style key* https://github.com/status-im/status-mobile/blob/8a869be01c728de5a444741db0fadc4aa6f4f3f5/doc/new-guidelines.md#always-add-styles-inside-the-style-key